src/java.base/unix/native/libnet/NetworkInterface.c

```diff
@@ -37,7 +37,7 @@
 #include <strings.h>
 #endif
 
-#if defined(_ALLBSD_SOURCE)
+#if defined(__APPLE__)
```

Comment:
Is this unnecessary for everything except MacOS or should this just have excluded OpenBSD?

Reply:
Just below the __APPLE__ section there's a complete _BSDONLY_SOURCE section coving the BSD's. If you prefer, I can merge the two into one _ALLBSD_SOURCE section. FreeBSD and __APPLE_ are the same, Open and NetBSD are slightly different. Either way is fine with me. Just let me know your preference.

Reply:
Got it.  I just needed a bit more context.  I think either version is fine.  I'd go with the one most closely matching upstream.